docs/faq.rst

```diff
@@ -114,18 +114,29 @@ make sure to output the denoised time series into a separate directory.
 
 .. _RICA: https://github.com/ME-ICA/rica
 
-*************************************************************************************
-[tedana] What is the difference between the kundu and minimal decision trees?
-*************************************************************************************
+.. _tree differences:
+
+********************************************************************************************
+[tedana] What is the difference between the tedana_orig, meica, and minimal decision trees?
```

Comment:
```suggestion
[tedana] What are the difference between the tedana_orig, meica, and minimal decision trees?
```

Reply:
Agh, I forgot to add an S at the end of `difference`.

Reply:
Fixed the typo & added the "s"